src/lib.rs

```diff
@@ -86,3 +86,4 @@ pub use util::key::PrivateKey;
 pub use util::key::PublicKey;
 pub use util::decimal::Decimal;
 pub use util::decimal::UDecimal;
+pub use util::bip158;
```

Comment:
We don't do this for any other modules in `util`, so I don't think you should. Users can do `use bitcoin::util::bip158` and then use `bip158::BlockFilter` etc.